edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/DependencyManager.cs

```diff
@@ -79,6 +79,7 @@ public void Register(ContainerBuilder builder)
                 });
 
             bool optimizeForPerformance = this.configuration.GetValue("OptimizeForPerformance", true);
+            bool usePooledBuffers = this.configuration.GetValue("UsePooledBuffers", false);
```

Comment:
Can we make this part of MqttSettings - https://github.com/Azure/iotedge/blob/master/edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Service/appsettings_hub.json#L2
So something like - mqttSettings__UseBufferPooling or mqttSettings__UsePooledBuffers